compose.yaml

```diff
@@ -13,7 +13,7 @@ services:
   app:
     build: ./
     volumes:
-      - ./application:/data
+      - ./app:/app
       - ./auth.json:/root/.composer/auth.json
```

Comment:
This would be a good candidate for [extensions](https://docs.docker.com/reference/compose-file/extension/).